articles/mfa/concepts/mfa-api.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Multi-factor Authentication API
+description: Overview of available multi-factor authentication APIs
+topics:
+  - mfa
+  - mfa-api
+contentType:
+  - index
+useCase:
+  - customize-mfa
+---
+
+# Multi-factor Authentication API
+
+The <dfn data-key="multifactor-authentication">Multi-factor Authentication (MFA)</dfn> API endpoints allow you to enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token), as well enroll and manage MFA factors.
+
+## Multi-factor authentication with the Token endpoint
+
+The Token endpoints cover the following use cases:
```

Comment:
endpoint or endpoints? The above and below of this go back and forth on plurality